app/scripts/models/reliers/oauth.js

```diff
@@ -159,7 +159,13 @@ define([
           //jshint camelcase: false
           // server version always takes precedent over the search parameter
           self.set('redirectUri', serviceInfo.redirect_uri);
+          self.set('termsUri', serviceInfo.terms_uri);
+          self.set('privacyUri', serviceInfo.privacy_uri);
+          self.set('trusted', serviceInfo.trusted);
```

Comment:
Note that `privacy_uri` and `terms_uri` are allowed to be empty strings.  It's not clear what we should do in this case, e.g. should it suppress display of the "I agree" block, or just not turn the text into hyperlinks, or something else?